Packs/Legacy/Integrations/Panorama/Panorama.py

```diff
@@ -2228,27 +2228,38 @@ def panorama_get_url_category_command(url_cmd: str):
     urls = argToList(demisto.args()['url'])
 
     categories_dict: Dict[str, list] = {}
+    categories_dict_hr: Dict[str, list] = {}
```

Comment:
not sure why do we need it if we have categories_dict

Reply:
I want to have a separate one for the HR, as `categories_dict` is used to populate the data from the context, so we can add new urls to already pre defined categories in the context.